### PR TITLE
Update import-physical-badging-data.md

### PR DESCRIPTION
**Pull request by NarraSaiKrishna**
Updated fields description

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/import-physical-badging-data.md [hunk 1]
```diff
@@ -228,7 +228,7 @@ After you create the physical badging connector and push your physical badging d
 
    ![Physical badging connector log file displays number of objects from the JSON file that were uploaded.](..\media\PhysicalBadgingConnectorLogFile.png)
 
-   The **RecordsSaved** field indicates the number of objects in the JSON file that uploaded. For example, if the JSON file contains four objects, then the value of the **RecordsSaved** fields is 4, if the script successfully uploaded all the objects in the JSON file.
+   The **RecordsSaved** field indicates the number of records in the JSON file that got uploaded. For example, if the JSON file contains four records, then the value of the **RecordsSaved** fields is 4, if the script successfully uploaded all the records in the JSON file. The **RecordsSkipped** field indicates the number of records in the JSON file that got skipped. Before uploading records in the JSON file, Email Ids of the records will be validated. Any record with invalid Email Id will be skipped and the corresponding Email Id is displayed in the field **EmailIdsNotSaved**
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/import-physical-badging-data.md]:
```suggestion
   The **RecordsSaved** field indicates the number of records in the JSON file that got uploaded. For example, if the JSON file contains four records, then the value of the **RecordsSaved** fields is 4 if the script successfully uploaded all the records in the JSON file. The **RecordsSkipped** field indicates the number of records in the JSON file that got skipped. Before uploading records in the JSON file, the Email IDs of the records will be validated. Any record with an invalid Email ID will be skipped and the corresponding Email ID is displayed in the field **EmailIdsNotSaved**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NarraSaiKrishna, kindly commit the suggested changes. Many thanks.

**Comment by NarraSaiKrishna** [comment]
@JohanFreelancer9 I have committed the changes. Let me know if I need to do anything else.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by NarraSaiKrishna** [comment]
@JohanFreelancer9 How long will it take to reflect in the document?

**Comment by JohanFreelancer9** [comment]
@NarraSaiKrishna, your changes will be reviewed by the docs team and if approved, will be merged. Unfortunately, I cannot give you a timeline.

**Comment by yogkumgit** [comment]
@v-tophillips Please merge this PR. Thanks

**Review by v-tophillips — APPROVED** [review]

**Pull request merged**